app.py

```diff
@@ -38,34 +44,46 @@
     }
 
     config = None
-
-    with open('config.yml', 'r') as config_file:
-        try:
-            config = yaml.load(config_file)
-        except yaml.YAMLError as err:
-            print(err)
-            exit(1)
-
+    config_path = None
```

Comment:
Initializing config_path is unnecessary since it will always be initialized in the if/else